services/core-api/app/api/now_applications/transmogrify_now.py

```diff
@@ -95,6 +95,7 @@ def _transmogrify_now_details(now_app, now_sub, mms_now_sub):
     now_app.work_plan = now_sub.descexplorationprogram
     now_app.merchantable_timber_volume = now_sub.timbertotalvolume
     now_app.proposed_annual_maximum_tonnage = now_sub.maxannualtonnage
+    now_app.adjusted_annual_maximum_tonnage = now_sub.maxannualtonnage
     now_app.is_access_gated = now_sub.isaccessgated == 'Yes'
```

Comment:
Discussed with Aaron to pre-populate the "adjusted_annual_maximum_tonnage" as the "proposed_annual_maximum_tonnage", users have the ability to edit it later. 
If this is not populated on the UI the logic that determines if the application is valid according to the fees paid will not run.